### DISCUSSION TITLE
Provide intersphinx objects.inv

### DISCUSSION THREAD

**Discussion by mschoettle** [encode/django-rest-framework#8376] [Ideas & Suggestions]
I started using `mkdocs` with `mkdocstrings` [1] and it has the ability to import `objects.inv` ([see intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)) files to reference elements from other projects. Unfortunately, DRF does not provide this file.

I am aware that you are using `mkdocs` and the aforementioned `mkdocstrings` actually generates the `objects.inv` file. So I am wondering whether there is interest to provide this. It would be great to be able to reference a code element in docstrings and have it link automatically to the relevant part in the DRF documentation.

[1] https://mkdocstrings.github.io/

**Comment by onegreyonewhite** [comment]
@mschoettle here is #9587
